RELEASE_NOTES.md

```diff
@@ -1,5 +1,21 @@
 # Auth Management UI Release Notes
 
+## 0.6
```

Comment:
this is steam release 0.5-dev for this component, please don't change release
without a prepare release management process